content/2_Bronze/Intro_DS.mdx

```diff
@@ -710,6 +709,261 @@ See [here](/general/notes-cpp-ds).
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Why are data structures useful?
+    <Quiz.Answer correct>
+      Data structures can provide efficient operations on certain types of data.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Data structures can store different types of data.
+      <Quiz.Explanation>
+        Correct. 
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      They provide a step-by-step set of rules to solve a problem.
+      <Quiz.Explanation>
+        Incorrect. This is an algorithm.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Data structures are sometimes inefficient.
+      <Quiz.Explanation>
+        Incorrect. Data structures are not useful for this reason.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+</Quiz>
+</CPPSection>
+<JavaSection>
+<Quiz>
+  <Quiz.Question>
+    How do you count the number of items in an `ArrayList`? Suppose we named it `list`.
+    <Quiz.Answer correct>
+      `list.size()`
+      <Quiz.Explanation>
+        Correct. `size()` returns the number of elements in the data structure!
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `list.length()`
+      <Quiz.Explanation>
+        Incorrect. The method `length()` does not exist on an `ArrayList`
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `list.count()` 
+      <Quiz.Explanation>
+        Incorrect. The method `count()` does not exist on an `ArrayList`
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `list.length`
+      <Quiz.Explanation>
+        Incorrect. The property `length` exists in arrays, but not `ArrayList`
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Why are data structures useful?
+    <Quiz.Answer correct>
+      Data structures can provide efficient operations on certain types of data.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Correct. Data structures can store different types of data.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      They provide a step-by-step set of rules to solve a problem.
+      <Quiz.Explanation>
+        Incorrect. This is an algorithm.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Data structures are sometimes inefficient.
+      <Quiz.Explanation>
+        Incorrect. Data structures are not useful for this reason.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+</Quiz>
+</JavaSection>
+<PySection>
+<Quiz>
+  <Quiz.Question>
+    How do you count the number of items in a `list`? Suppose we named the list `l`
+    <Quiz.Answer correct>
+      ```py
+	  len(l)
+	  ```
+      <Quiz.Explanation>
```

Comment:
Add some explanation here so that after submitting, something changes besides just the color.